pkg/restore-util/range.go

```diff
@@ -33,6 +36,47 @@ func (r *Range) contains(key []byte) bool {
 		(len(end) == 0 || bytes.Compare(key, end) < 0)
 }
 
+// sortRanges checks if the range overlapped and sort them
+func sortRanges(ranges []Range, rewriteRules *RewriteRules) ([]Range, error) {
+	rangeTree := NewRangeTree()
+	for _, rg := range ranges {
+		if rewriteRules != nil {
+			var startRule *import_sstpb.RewriteRule
+			rg.StartKey, startRule = replacePrefix(rg.StartKey, rewriteRules)
+			if startRule == nil {
+				log.Warn("cannot find rewrite rule", zap.ByteString("key", rg.StartKey))
```

Comment:
```suggestion
				log.Warn("cannot find rewrite rule", zap.Binary("key", rg.StartKey))
```

Don't use `ByteString` for bytes which is not UTF-8.